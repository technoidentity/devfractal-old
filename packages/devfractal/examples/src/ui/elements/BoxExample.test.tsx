import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { Box } from './BoxExample'

test('Box Example', () => {
  const component: ReactTestRenderer = renderer.create(<Box />)
  expect(component.toJSON()).toMatchSnapshot()
})
