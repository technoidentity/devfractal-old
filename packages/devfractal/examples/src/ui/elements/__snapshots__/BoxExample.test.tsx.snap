// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box Example 1`] = `
<section
  className="section"
>
  <div
    className="box"
  >
    <article
      className="media"
    >
      <figure
        className="media-left"
      >
        <figure
          className="image is-64x64"
        >
          <img
            className=""
            src="https://bulma.io/images/placeholders/128x128.png"
          />
        </figure>
      </figure>
      <div
        className="media-content"
      >
        <div
          className="content"
        >
          <p>
            <strong>
              John Smith
            </strong>
             
            <small>
              @johnsmith
            </small>
            <small>
              31m
            </small>
            <br />
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean efficitur sit amet massa fringilla egestas. Nullam condimentum luctus turpis.
          </p>
        </div>
      </div>
    </article>
  </div>
</section>
`;
